samples/fuel_gauge/lc709203f/README.rst

```diff
@@ -0,0 +1,45 @@
+.. zephyr:code-sample:: max17048
+   :name: MAX17048 Li-Ion battery fuel gauge
```

Comment:
You have copy-paste typos plus I'd rather you turn the existing fuel gauge sample into a generic one as there is nothing specific to either driver in it
I.e. have the sample expect a fuel_gauge0 alias or something like that

Reply:
`fuel_gauge_get_props` just seems to be a convenience wrapper around `fuel_gauge_get_prop` so I think it might be worth changing the code so that you "try" all the props?
Or maybe even better, since it's a sample meant to demonstrate how to use the API, showcase a mix of `fuel_gauge_get_props` and `fuel_gauge_get_prop` -- `fuel_gauge_get_props` for FUEL_GAUGE_RELATIVE_STATE_OF_CHARGE and FUEL_GAUGE_VOLTAGE (if your confident 100% of the fuel gauge will always have these) and then try `fuel_gauge_get_prop` one by one on some of the "bonus" properties?

Reply:
so actually similar to the tests. 
I think I will do a first iteration where it is the basic sample only with these, to have it at least working properly:
```
fuel_gauge_prop_t props[] = {
	FUEL_GAUGE_RELATIVE_STATE_OF_CHARGE,
	FUEL_GAUGE_VOLTAGE,
};
```
and next week or when I have time again, extend the sample as you suggested, because I like your idea very much.